pulp_smash/resources/__init__.py

```diff
@@ -0,0 +1,4 @@
+# coding=utf-8
+"""Modules for easy manipulation with pulp resources like users, repositories
+etc."""
```

Comment:
We can word-smith this and make it fit on one line. How about:

    """Modules for manipulating pulp resources like users, repositories, etc."""